client/state/terms/schema.js

```diff
@@ -16,7 +16,8 @@ export const termsSchema = {
 								slug: { type: 'string' },
 								description: { type: 'string' },
 								post_count: { type: 'number' },
-								parent: { type: 'number' }
+								parent: { type: 'number' },
+								meta: { type: 'object' }
```

Comment:
Also was getting warnings about the missing `meta` option in this Object so quick fix for that here as well.